### PR TITLE
v1.1 changes and other minor changes

### PR DESCRIPTION
**Pull request by prashant-shahi**
- changed uid to [uid]. #73 
- increased the maximum height of %expandable% block - resolving the partial visibility of query in /intro/4 #71 
- fixed previous page reference in markdowns
- replaced capture_output with the native implementation for support of python<=3.7

### CODE REVIEW

File: content/moredata/1.txt [hunk 1]
```diff
@@ -18,7 +18,7 @@ type Genre {
 
 # Define Directives and index
 
-director.film: uid @reverse .
-genre: uid @reverse .
+director.film: [uid] @reverse .
+genre: [uid] @reverse .
```
Comment (MichelDiz) [inline content/moredata/1.txt:22]: Some of this is covered here https://github.com/dgraph-io/tutorial/pull/53/files Not sure if it's okay to overwrite tho.

Comment (prashant-shahi) [reply content/moredata/1.txt:22]: Okay, I will be merging all the latest changes.

Comment (paulftw) [reply content/moredata/1.txt:22]: so what have you guys decided? are we changing schema in this PR or in 53?

**Comment by prashant-shahi** [comment]
Some of the changes are covered and there seem to be quite a lot of changes done since then.

Let me find out what and all changes are needed from the current master and then send another PR.

**Pull request closed without merging**